src/otx/engine/engine.py

```diff
@@ -312,6 +313,10 @@ def test(
             datamodule = self._auto_configurator.update_ov_subset_pipeline(datamodule=datamodule, subset="test")
             model = self._auto_configurator.get_ov_model(model_name=str(checkpoint), label_info=datamodule.label_info)
 
+        # NOTE: Re-initiate datamodule without tiling as model API supports its own tiling mechanism
+        if isinstance(model, OVModel) and isinstance(datamodule.subsets["test"], OTXTileDataset):
+            datamodule = self._auto_configurator.update_ov_subset_pipeline(datamodule=datamodule, subset="test")
```

Comment:
```suggestion
            model = self._auto_configurator.get_ov_model(model_name=str(checkpoint), label_info=datamodule.label_info)

        # NOTE: Re-initiate datamodule without tiling as model API supports its own tiling mechanism
        if isinstance(model, OVModel):
            datamodule = self._auto_configurator.update_ov_subset_pipeline(datamodule=datamodule, subset="test")
```
What about the suggestions above?
Wouldn't it be nice to separate model and datamodule updates to reduce duplication?

The same goes for the changes below.